js/test/CryptTool.js

```diff
@@ -38,15 +43,19 @@ describe('CryptTool', function () {
         // The below static unit tests are included to ensure deciphering of "classic"
         // SJCL based pastes still works
         it(
-            'supports PrivateBin v1 ciphertext (SJCL & browser atob)',
-            function () {
+            'supports PrivateBin v1 ciphertext 1 (SJCL & browser atob)',
```

Comment:
```suggestion
            'supports PrivateBin v1 ciphertext v1 (SJCL & browser atob)',
```

I wondered whether this 1 was a typo as you already mentioned a version before (And could have accidentially doubled it)? Maybe clarify the code like this?

Reply:
hopefully fixed by clarifying what these two split tests are about - one uses a password, the other is without password and we expect both types to still decrypt in our current code